.github/workflows/ci.yml

```diff
@@ -64,3 +65,16 @@ jobs:
           python3 -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz?run=${{ env.RUN }}', 'builder')"
           chmod a+x builder
           ./builder build -p ${{ env.PACKAGE_NAME }} --target=${{ matrix.target }}-armv8
+
+  check-submodules:
+    runs-on: ubuntu-20.04 # latest
+    steps:
+      - name: Checkout Source
+        uses: actions/checkout@v2
```

Comment:
use latest v3?

might be worth switching to latest ubuntu lts - 22.04 as well

Reply:
Thanks, fixed.